koku/koku/configurator.py

```diff
@@ -500,26 +502,46 @@ def get_object_store_region(requested_name: str = ""):
     @staticmethod
     def get_database_name():
         """Obtain database name."""
+        use_readreplica = ClowderConfigurator._use_read_replica()
+        if use_readreplica:
+            with open("/etc/db/readreplica/db_name") as file:
+                return file.read().rstrip()
```

Comment:
`pathlib` makes reading the contents of a file a bit more succinct. I'm not sure we need the local variable unless it's there to increase legibility.
```suggestion
        if ClowderConfigurator._use_read_replica():
            try:
                return pathlib.Path("/etc/db/readreplica/db_name").read_text().rstrip()
            except FileNotFoundError:
                # maybe log something here
```

Reply:
updates made.